crates/uv-cli/src/lib.rs

```diff
@@ -1955,6 +1955,15 @@ pub struct BuildArgs {
     #[arg(value_parser = parse_file_path)]
     pub src: Option<PathBuf>,
 
+    /// Build a specific package in the workspace.
+    ///
+    /// The workspace will be discovered from the provided source directory, or the current
+    /// directory if no source directory is provided.
+    ///
+    /// If the workspace member does not exist, uv will exit with an error.
+    #[arg(long)]
+    pub package: Option<PackageName>,
```

Comment:
I'm not a huge fan of the interaction between `src` and `package` because I feel like it shows some warts between an API that's designed for projects vs. a general-purpose API.

Reply:
I don't mind the interaction as described.